libs/core/langchain_core/utils/pydantic.py

```diff
@@ -64,3 +128,106 @@ def wrapper(cls: Type[BaseModel], values: Dict[str, Any]) -> Dict[str, Any]:
         return func(cls, values)
 
     return wrapper
+
+
+def _create_subset_model_v1(
```

Comment:
Unfortunately this function fails for pydantic v1 models that use `Annotated` with `Field`, e.g.

```python
class InputModel(BaseModel):
    query: Annotated[str, pydantic_v1.Field(description="Hello World")]

_create_subset_model_v1("test", InputModel, InputModel.__annotations__.keys())
```

This produces the following error:

```plain
ValueError: cannot specify `Annotated` and value `Field`s together for 'query'
```